include/OpenSpaceToolkit/Astrodynamics/Trajectory/Propagator.hpp

```diff
@@ -41,6 +42,7 @@ namespace trajectory
 using ostk::core::ctnr::Array;
 using ostk::core::types::Integer;
 using ostk::core::types::Real;
+using ostk::core::types::Shared;
```

Comment:
```suggestion
```

idem. I might be blind, but don't see a Shared pointer added

Reply:
This is used by the dynamics class in the constructor that already exists.
We get away with it because we're importing a file that has the `using` defined but ideally we should be explicit per file.